src/modules/course/dto/course-update.dto.ts

```diff
@@ -129,4 +138,50 @@ export class UpdateCourseDTO {
   @IsUrl()
   @IsNotEmpty()
   courseTrailerUrl: string;
+
+  /**
+   * The discounted price of the course
+   * @example 120
+   */
+  @IsNumber()
+  crossPrice: number;
+
+  /**
+   * The short description of the course
+   * @example 'Short description--'
+   */
+  @IsString()
+  courseShortDescription: string;
+
+  /**
+   * The long description of the course
+   * @example 'Long description--'
+   */
+  @IsString()
+  courseLongDescription: string;
+
+  /**
+   * The rating of the course
+   * @example 5
+   */
+  @IsNumber()
+  rating: number;
+
+  /**
+   * The prerequisites of the course
+   * @example ["HTML","CSS"]
+   */
+  prerequisites: string[];
+
+  /**
+   * The skills of the course
+   * @example ["HTML","CSS"]
+   */
+  skills: string[];
+
+  /**
+   * what will one learn from the course
+   * @example ["HTML","CSS"]
```

Comment:
Provide at least 4 -5 sentence example so that the end user could take the guess.

Like "You will get to know about web technologies basics" "A good understanstanding of Html, css and JS"